components/lib/crash/src/test/java/mozilla/components/lib/crash/service/GleanCrashReporterServiceTest.kt

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.crash.service
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.work.testing.WorkManagerTestInitHelper
+import mozilla.components.lib.crash.Crash
+import mozilla.components.lib.crash.GleanMetrics.CrashMetrics
+import mozilla.components.service.glean.Glean
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Assert.assertFalse
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.verify
+import org.mockito.Mockito.verifyNoMoreInteractions
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class GleanCrashReporterServiceTest {
+    private val context: Context
+        get() = ApplicationProvider.getApplicationContext()
+
+    private fun setupGleanCrashReporterServiceTest() {
+        WorkManagerTestInitHelper.initializeTestWorkManager(
```

Comment:
Eugh, it hurts me so much we need to do this in every Glean consumer :(

Reply:
Totally agree.  I wonder if we could create another testAPI function that would perform this dirty work.  Maybe inside of the `enabledTestingMode` function?

Reply:
That would be weird, as we'd need to depend on the workmanager testing package in a non testing environment :( Anyway, there's [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1552114) for it, let's brainstorm there!